src/features/events_list/components/CurrentEventFromResource/CurrentEventFromResource.tsx

```diff
@@ -0,0 +1,34 @@
+import { useAtom } from '@reatom/react';
+import { ErrorMessage } from '~components/ErrorMessage/ErrorMessage';
+import { LoadingSpinner } from '~components/LoadingSpinner/LoadingSpinner';
+import { i18n } from '~core/localization';
+import { currentEventResourceAtom } from '~features/current_event/atoms/currentEventResource';
+import { createStateMap } from '~utils/atoms';
+import { EpisodeTimelineToggle } from '../EpisodeTimelineToggle/EpisodeTimelineToggle';
+import { EventCard } from '../EventCard/EventCard';
+
+export function CurrentEventFromResource({ hasTimeline }: { hasTimeline?: boolean }) {
```

Comment:
Just CurrentEvent - postfix `resource` it for atoms that load something from api